Makefile

```diff
@@ -25,3 +26,39 @@ ifndef GOLANGCILINT
 	@gobin github.com/golangci/golangci-lint/cmd/golangci-lint@v1.17.1
 endif
 	@golangci-lint run
+
+#################
+# Swagger stuff #
+#################
+
+PWD := $(shell pwd)
+GOPATH := $(shell go env GOPATH)
+
+gen: api_docs swagger
+
+api_docs:
+	@echo "Installing swagger-merger" && npm install swagger-merger -g
```

Comment:
npm is a new dev dependency, please help add a section about it in README, thanks

Reply:
:+1: